docs/development/component-checklist.md

```diff
@@ -93,8 +93,8 @@ This document provides a checklist for them that you can walk through.
 
 6. **Choose the proper Seccomp profile** ([example 1](https://github.com/gardener/gardener/blob/b0de7db96ad436fe32c25daae5e8cb552dac351f/pkg/component/nodelocaldns/nodelocaldns.go#L283-L287), [example 2](https://github.com/gardener/gardener/blob/b0de7db96ad436fe32c25daae5e8cb552dac351f/pkg/component/nginxingress/nginxingress.go#L447))
 
-   The [Seccomp profile](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-seccomp-profile-for-a-container) will be defaulted by `gardener-resource-manager`'s SeccompProfile webhook which works well for the majority of components.
-   However, in some special cases you might need to overwrite it.
+   For components running in the Seed cluster, the [Seccomp profile](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-seccomp-profile-for-a-container) will be defaulted to `RuntimeDefault` by `gardener-resource-manager`'s SeccompProfile webhook which works well for the majority of components. However, in some special cases you might need to overwrite it.
+   For components running in the Shoot cluster, the `gardener-resource-manager`'s SeccompProfile webhook is not enabled. It is recommended components deployed in the Shoot cluster to explicitly set their Seccomp profile to `RuntimeDefault` (the default container runtime Seccomp profile), if possible because otherwise, the default behavior is `Unconfined` (no Seccomp profile is applied).
```

Comment:
```suggestion
   For components running in the Shoot cluster, the `gardener-resource-manager`'s SeccompProfile webhook is not enabled. If possible, it is recommended to explicitly set the Seccomp profile to `RuntimeDefault` (the default container runtime Seccomp profile) for components deployed in the Shoot cluster. Otherwise, the default behavior is `Unconfined` (no Seccomp profile is applied).
```